osu.Game/Overlays/Settings/Sections/Audio/VolumeSettings.cs

```diff
@@ -12,11 +13,12 @@ public class VolumeSettings : SettingsSubsection
         protected override string Header => "Volume";
 
         [BackgroundDependencyLoader]
-        private void load(AudioManager audio)
+        private void load(AudioManager audio, OsuConfigManager config)
         {
             Children = new Drawable[]
             {
                 new SettingsSlider<double> { LabelText = "Master", Bindable = audio.Volume, KeyboardStep = 0.1f },
+                new SettingsSlider<double> { LabelText = "Master (Window Inactive)", Bindable = config.GetBindable<double>(OsuSetting.VolumeInactive), KeyboardStep = 0.1f },

```

Comment:
Not sure that "Window Inactive" does fit here since the game can be runned fullscreen.

Reply:
a full screen app is still a window in windows. but sure, maybe we can rename to "Game Inactive".

Reply:
I'm happing leaving it as window inactive.